services/QuillLMS/db/migrate/20210423165423_change_avg_completion_time_column.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeAvgCompletionTimeColumn < ActiveRecord::Migration
+  def change
+    remove_column :activity_healths, :avg_completion_time, :time
+    remove_column :activity_healths, :recent_assignments, :integer
+    add_column :activity_healths, :avg_mins_to_complete, :float
+    add_column :activity_healths, :flag, :string
+    add_column :activity_healths, :recent_plays, :integer
```

Comment:
I will do that for the `recent_plays` column. I can't do that for `avg_mins_to_complete` thoough because that column can't change from a time zone data type to a float, so I have to do a remove-and-add.